js/templates/modals/settings/advanced/smtpintegration.html

```diff
@@ -0,0 +1,120 @@
+<form class="box padKids padStack clrP clrBr js-smtpintegration">
+  <div class="flexRow">
+    <h3><%= ob.polyT('settings.smtpIntegration') %></h3>
+  </div>
+  <div class="flexRow">
+    <div class="col3">
+      <label><%= ob.polyT('settings.smtpNotifications') %></label>
+    </div>
+    <div class="col9">
+      <div class="btnStrip">
+        <div class="btnRadio clrBr">
+          <input type="radio"
+                 name="smtpNotifications"
+                 value="true"
+                 id="smtpNotificationsOn"
+                 data-var-type="boolean"
+          <% if(ob.smtpNotifications) { %>checked<% } %>>
+          <label for="smtpNotificationsOn"><%= ob.polyT('settings.on') %></label>
+        </div>
+        <div class="btnRadio clrBr">
+          <input type="radio"
+                 name="smtpNotifications"
+                 value="false"
+                 id="smtpNotificationsOff"
+                 data-var-type="boolean"
+          <% if(!ob.smtpNotifications) { %>checked<% } %>>
+          <label for="smtpNotificationsOff"><%= ob.polyT('settings.off') %></label>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="flexRow">
+    <div class="col3">
+      <label for="smtpServerAddress">
+        <%= ob.polyT('settings.smtpServerAddress') %>
+      </label>
+    </div>
+    <div class="col9">
+      <input class="clrBr" 
+        required
+        pattern=".*"
```

Comment:
Re: validation. This is done at the template level. It's a matter of:

1.) Pass in you model errors when rendering the template:

```
      this.$el.html(t({
        errors: this.model.validationError || {},
      });
```

Any nested attributes will be included in the validationError object, but they'll be flattened, for example:

```
{
  email: 'bull-shiznit-email',
  nestedModel: {
    email: 'crap-on-a-stick-yo',
  }
}
```

would result in the following error object:

```
{
  email: ['The email is not valid'],  // errors are returned in an array because a field could fail multiple          validations
  nestedModel.email: ['The email is not valid'],
}
```

2.) If a field has an error, render the `formErrorTmpl` (it's by default going to available in your template) above it's corresponding field, e,g:

```
        <% if (ob.errors.email) print(ob.formErrorTmpl({ errors: ob.errors.email })) %>
        <input type="text" name="email" value="<%= ob.email %>">

        <section>  /// nested model
          <% if (ob.errors['nestedModel.email']) print(ob.formErrorTmpl({ errors: ob.errors['nestedModel.email'] })) %>
          <input type="text" name="nestedModel.email" value="<%= ob.nestedModel.email %>">
       </section>
```

3.) Make sure that at the end of you're save() method you render() (this way any new errors will be displayed and any old ones will be cleared), e.g:

```
    this.render();
    // next two lines will scroll to the first error
    const $firstErr = this.$('.errorList:first'); 
    if ($firstErr.length) $firstErr[0].scrollIntoViewIfNeeded();
```

Please see Page.js and / or EditListing.js (and their templates) as an example (FWIW, I would start with Page.js, since EditListing.js is much more complex (very deeply nested)).